python/tvm/relay/frontend/mxnet.py

```diff
@@ -903,6 +903,22 @@ def _mx_resize(inputs, attrs):
     return _op.image.resize(inputs[0], size,
                             coordinate_transformation_mode="align_corners")
 
+def _mx_amp_multicast(inputs, attrs):
+    cast_narrow = attrs.get_bool("cast_narrow", False)
+    dtypes = [_infer_type(x).checked_type.dtype for x in inputs]
+    supported_dtypes = ['float16', 'float32']
+    assert all([x in supported_dtypes for x in dtypes]), \
+            "amp_multicast support is limited to float16 and float32 inputs only."
+    dtype = 'float32' if cast_narrow else dtypes[0]
```

Comment:
```suggestion
    dtype = dtypes[0]
```

Reply:
Just a minor and soft suggestion, so feel free to ignore.  Maybe following has better readability

~~~
has_float16 = any(x == "float16" for x in dtypes)
has_float32 = any(x == "float32" for x in dtypes)

# logic with three vars - cart_narrow, has_float16, has_float32
~~~